src/components/panes/EditPersonTagPane.jsx

```diff
@@ -73,4 +89,10 @@ export default class EditPersonTagPane extends PaneBase {
         this.props.dispatch(updatePersonTag(tagId, values));
         this.closePane();
     }
+
+    onValueChange(name, value) {
+        if (name === "title") {
+            this.setState({ shouldShowButton: Boolean(value) })
```

Comment:
Nice and simple!

Another way of doing `Boolean(value)` is to do `!!value`, which you will find in other places of this codebase.